src/scripts/ui/inline-roll-links.ts

```diff
@@ -66,7 +67,7 @@ export const InlineRollLinks = {
                 : null;
         };
 
-        $repostLinks.filter("i[data-pf2-repost]").on("click", (event) => {
+        $html.find("i[data-pf2-repost]").on("click", (event) => {
```

Comment:
We're getting off of JQuery. Try to find where we have htmlQuery() and htmlQueryAll() and do it as it says there.

It will be more verbose but the less stuff we have to change later on once the JQuery death knell is sent out the better.

Reply:
Done!